pyms/cmd/main.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals, print_function
+
+import argparse
+import sys
+
+from pyms.utils.crypt import Crypt
+
+
+class Command:
```

Comment:
The `Command` class will only be to encrypt and decrypt? With that name I'd expect it to be multi-purpose.

Reply:
keep calm :smile: this is the first command, we have in the roadmap other options like generate config, generate a scaffold... :wink:

Reply:
Nice then x)